llama_stack/ui/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
```

Comment:
can you add a blurb here at the top which says "## This is Work in Progress" and that "this UI will spin up when users run `llama stack run`" etc.?